### DISCUSSION TITLE
Use Pluto as dashboarding solution for multiple users

### DISCUSSION THREAD

**Discussion by Sh4pe** [fonsp/Pluto.jl#2594] [Questions about using Pluto]
I had a look at [Visualization Dashboards with Pluto!](https://pretalx.com/juliacon-2022/talk/SQJTRS/) which shows how you can turn Pluto notebooks into dashboards. While it seems that this uses quite a bit of implementation details of Pluto that are not yet public interfaces (e.g. `PlutoUI.ExperimentalLayout` at 5:43), I really like the approach shown there. It would be a solution for my use-case of showing dashboards to internal users that to not care about the cell code (or that should not see it).

However, I'd want to deploy the dashboard to a server so that users can e.g. upload their data to analyze and visualize. That leads me to the problem of multiple users and multiple session.

As far as I see it, when I run Pluto somewhere, there is no notion of different sessions or different users. So do you see any way to turn this into a dashboard that could be used independently by multiple users at the same time? I'm not talking about many users, but it would be nice if my solution would not fall apart once two users use it simultaneously. :)

**Comment by fonsp** [comment]
This sounds like it might be a good use case for https://github.com/JuliaPluto/PlutoSliderServer.jl, if you use the PlutoUI file upload input! But only if the datasets are <10MB or so, since they will be sent along with every request.

Check out the plutocon video linked on the PlutoSliderServer readme for more info about multiple users/sessions.

**Reply by Sh4pe** [reply]
Thank you very much for the suggestion and sorry for not answering. I'll have a look into it!

**Reply by fonsp** [reply]
Hey @Sh4pe ! How did it go?
